modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/display/context/AnalyticsReportsDisplayContext.java

```diff
@@ -195,6 +197,19 @@ public String getLiferayAnalyticsURL(long companyId) {
 			"authorName",
 			_analyticsReportsInfoItem.getAuthorName(
 				_analyticsReportsInfoItemObject)
+		).put(
+			"authorPortraitURL",
+			() -> {
+				if ((_user == null) || (_user.getPortraitId() == 0)) {
```

Comment:
What do you think about using 
Optional.ofNullable(_user).filter (_user.getPortraitId() > 0).map(user -> user.getPortraitURL(_themeDisplay))

Reply:
I love it, let's stop using null checks ;)